app/ui/src/main/java/com/fsck/k9/fragment/MessageListFragment.java

```diff
@@ -1,18 +1,6 @@
 package com.fsck.k9.fragment;
 
 
-import java.util.ArrayList;
```

Comment:
Commit 676eeeab10884456a5f70ce708a8aa5116ecbaf7 contains a lot of unrelated code style changes. Please get rid of these.

Reply:
done